docs/usage/schedule_deployment.md

```diff
@@ -0,0 +1,40 @@
+# Schedule the deployment
+
+You can schedule the deployment through QGIS Deployment Toolbelt using a task scheduler.
+
+## On Windows
+
+1. Open the **Task Scheduler**
+
+![Task scheduler](../static/task_scheduler_windows_app.png)
```

Comment:
Two thoughts (applicable to other images below):

- to respect ordered list "natural" numering, you have to indent your paragraph between points. Try it with 1. numbering only.
- use image path from website base URL instead of relative path

```suggestion
    ![Task scheduler](/static/task_scheduler_windows_app.png)
```